apps/auth/config.py

```
@@ -0,0 +1,71 @@
+    raise ValueError(
+        "\033[31m[ERROR]\033[0m \033[107mAUTH_KEY not set, please set it in the .env file\033[0m"
+    )
+AUTH_KEY = os.getenv("AUTH_KEY")
+if not os.getenv("ISSECURE"):
+    print("\033[33m[WARN]\033[0m ISSECURE not set, using default value of False")
+try:
+    ISSECURE = bool(os.getenv("ISSECURE", False))
+except ValueError:
+    raise ValueError(
+        "\033[31m[ERROR]\033[0m \033[107mISSECURE must be a boolean value, please set it in the .env file\033[0m"
+    )
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)